content/2018-munich/speakers/richard-hartmann.md

```diff
@@ -4,4 +4,4 @@ title: Richard Hartmann
 
 ## Richard Hartmann
 
-Richard is a Prometheus team member, currently focusing on building a new datacenter for SpaceNet, Richard comes from the networking side of things and has been on call 24/7 for over a decade in his last life. He learned the hard way that precise monitoring which supports operations is key to keeping people sane, and allowing them both sleep and a social life.
+Richard is a Prometheus team member, currently focusing on building a new datacenter for SpaceNet. He comes from the networking side of things and has been on call 24/7 for over a decade in his last life. He learned the hard way that precise monitoring which supports operations is key to keeping people sane, and allowing them both sleep and a social life.
```

Comment:
past life?

Reply:
It's the one directly before this life; but I am ambivalent.

Reply:
Last life would imply reincarnation usually.

Reply:
That's pretty much the point, yes.